package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "sass-embedded",
   "version": "1.87.0",
-  "protocol-version": "3.1.0",
+  "protocol-version": "3.2.0",
```

Comment:
I believe the repo is currently in a broken state because https://github.com/sass/sass/pull/4073 was submitted bumping the protocol version, and the repo expects this to match.

Reply:
fix: https://github.com/sass/embedded-host-node/pull/372